client/src/utils/calendarUtils.js

```diff
@@ -0,0 +1,69 @@
+import moment from "moment";
```

Comment:
We're using date-fns in other parts of the app for time formatting, might consider changing if there's nothing specific to moment you need